test/system/090-events.bats

```diff
@@ -355,7 +355,15 @@ EOF
     local cname=c-$1-$(safename)
     t0=$(date --iso-8601=seconds)
 
-    CONTAINERS_CONF_OVERRIDE=$containersConf run_podman create --name=$cname $IMAGE
+    # Create a base image, airgapped from $IMAGE so this test is
+    # isolated from tag/label changes.
+    baseimage=i-$1-$(safename)
+    tmpcname=c-tmp-$1-$(safename)
+    run_podman run --name $tmpcname $IMAGE true
```

Comment:
I assume this could just be create without run which should make it faster as a run is much slower.

Reply:
Ouch - of course. Fixed, thank you.